python/tests/estimators/test_keras_estimators.py

```diff
@@ -105,7 +103,27 @@ def setUp(self):
     def tearDown(self):
         shutil.rmtree(self.temp_dir, ignore_errors=True)
 
+    def test_validate_params(self):
+        """Test that `KerasImageFileEstimator._validateParams` method works as expected"""
```

Comment:
we probably could be more thorough in making sure all the intersect/diff logic works but I think it's good enough.